cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,182 @@
+/// <reference types="cypress" />
+
 describe('Web Tables page', () => {
-  it('', () => {
+  const worker = {
+    firstName: 'Danylo',
+    lastName: 'Plutenko',
+    email: 'danyloplutenko@qa.team',
+    age: 21,
+    salary: 10000,
+    department: 'QA'
+  };
+
+  const changedWorker = {
+    firstName: 'Goofy',
+    lastName: 'Goober',
+    email: 'goofygoober@qa.team',
+    age: 18,
+    salary: 70000,
+    department: 'QC',
+  };
+
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables')
+  });
+
+  it('should have pagination', () => {
+    cy.get('.-pagination')
+      .should('contain', 'Previous')
+      .and('contain', 'Next');
+    
+    cy.contains('.-pageInfo', 'Page')
+      .should('exist');
+  });
+
+  it('should have rows count selection', () => {
+    cy.get('select')
+      .select('5 rows');
+
+    cy.get('select')
+      .should('contain', '5 rows');
+  });
+
+  it('should add new worker', () => {
+    cy.get('#addNewRecordButton')
+      .click();
+
+    cy.get('#firstName')
+      .type(worker.firstName);
+
+    cy.get('#lastName')
+      .type(worker.lastName);
+
+    cy.get('#userEmail')
+      .type(worker.email);
+
+    cy.get('#age')
+      .type(worker.age);
+
+    cy.get('#salary')
+      .type(worker.salary);
+
+    cy.get('#department')
+      .type(worker.department);
+
+    cy.get('#submit')
+      .click();
+
+    cy.get('.rt-tbody')
+      .should('contain', worker.firstName)
+      .and('contain', worker.lastName)
+      .and('contain', worker.email)
+      .and('contain', worker.age)
+      .and('contain', worker.salary)
+      .and('contain', worker.department);
+  });
+
+  it('should delete a worker', () => {
```

Comment:
try to implement a loop here

Reply:
i don't understand

Reply:
in the test "delete all workers" you make the same action 3 times. Create a loop to make this actions inside, to create you code more readable and compact